workshops/views.py

```diff
@@ -937,7 +937,6 @@ def all_events(request):
 
 
 @login_required
-@permission_required('workshops.add_todoitem', raise_exception=True)
```

Comment:
I think, POST requests that are intentionally generated to simulate form submissions would now end up creating TODOs.
Is that right?